statannotations/stats/StatResult.py

```diff
@@ -58,6 +58,12 @@ def significance_suffix(self):
             return 'ns'
         return ""
 
+    @property
+    def is_significant(self):
+        if self._corrected_significance is False:
```

Comment:
You may indeed use  `_corrected_significance` here as we have no additional behavior in the `corrected_significance` getter function. If we ever add something, we'll have to ask ourselves if we need that code when calling `is_significant` or not.
(Since I'm adding another comment, I think we could move this method above `significance_suffix` just for the alphabetic ordering ;-))